src/main/java/com/alipay/remoting/AsyncContext.java

```diff
@@ -29,4 +29,10 @@ public interface AsyncContext {
      * @param responseObject response object
      */
     void sendResponse(Object responseObject);
+
+    /**
+     * send exception back
+     * @param ex response exception
+     */
+    void sendException(Throwable ex);
```

Comment:
For compatibility, `sendException` should add default implement. However bolt is still using jdk 1.6 as language level . How about upgrade to 1.8 instead? @chuailiwu

Reply:
I agree with @OrezzerO that adding a new method in one interface could cause backward compatibility issues. Is it possible we upgrade the jdk to 1.8?

Reply:
I thinks we upgrade the jdk to 1.8 is OK for most base components sdk support 1.8. 
If some corner case has problem, either adapt it  or continue to use the lower version .

Reply:
@chuailiwu ok，i will make changes according to the comments.